frontends/web/src/components/accountselector/accountselector.tsx

```diff
@@ -20,7 +20,7 @@ import Select, { components, SingleValueProps, OptionProps, SingleValue, Dropdow
 import { AccountCode, IAccount } from '../../api/account';
 import { Button } from '../forms';
 import Logo from '../icon/logo';
-import AppContext from '../../contexts/AppContext';
+import { AppContext } from '../../contexts/AppContext';
```

Comment:
Seems like there's a lot of changes - but some are due to changing imports of `AppContext` to named exports. 😄